src/main/java/com/google/sps/servlets/NewPostServlet.java

```diff
@@ -64,7 +64,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             commentId = DatastoreManager.insertCommentInDatastoreIfNonToxic(name, comment);
         } catch(ToxicCommentException e) {
             logger.error(e);
-            throw new ServletException("Error: " + e.getMessage(), e);
+            String redirect = (feedBool == true) ? "feed.html?name=" + URLEncoder.encode(repName) : 
```

Comment:
you can just do feedBool ? <...>
also what is feedBool? could you rename it to something that actually describes what it is.